plugins/miscellanea/snapcast/install.sh

```diff
@@ -0,0 +1,141 @@
+## SnapCast installation script
+echo "Installing SnapCast and its dependencies..."
+INSTALLING="/home/volumio/snapcast-plugin.installing"
+
+if [ ! -f $INSTALLING ]; then
+
+	touch $INSTALLING
+	apt-get update
+
+	# Download latest SnapCast packages
+	mkdir /home/volumio/snapcast
+	wget $(curl -s https://api.github.com/repos/badaix/snapcast/releases/latest | grep 'armhf' | cut -d\" -f4) -P /home/volumio/snapcast
```

Comment:
Here is where we need to download a different file for x86 arch, correct? Is the idea to detect the system architecture here and then download the matching file for either rpi (armhf) or x86? And where else do we need to make changes for x86 exactly?

Reply:
@Saiyato I have no idea, sorry.  The only way I can help here is with the spare x86 laptop sitting right on my desk. Happy to test if your old laptop is bricked.